tests/drivers/nrfx_integration_test/Kconfig

```diff
@@ -12,13 +12,13 @@ config NRFX_ALL_DRIVERS
 	select NRFX_ADC		if HAS_HW_NRF_ADC
 	select NRFX_CLOCK	if HAS_HW_NRF_CLOCK && !CLOCK_CONTROL_MPSL
 	select NRFX_COMP	if HAS_HW_NRF_COMP
-	select NRFX_DPPI	if HAS_HW_NRF_DPPIC
 	select NRFX_EGU0	if HAS_HW_NRF_EGU0
 	select NRFX_EGU1	if HAS_HW_NRF_EGU1
 	select NRFX_EGU2	if HAS_HW_NRF_EGU2
 	select NRFX_EGU3	if HAS_HW_NRF_EGU3
 	select NRFX_EGU4	if HAS_HW_NRF_EGU4
 	select NRFX_EGU5	if HAS_HW_NRF_EGU5
+	select NRFX_GPPI
 	select NRFX_GPIOTE0	if HAS_HW_NRF_GPIOTE0
```

Comment:
nit: GPPI should be placed after GPIOTEs

Reply:
fixed